.github/workflows/nightly.yaml

```diff
@@ -48,16 +48,16 @@ jobs:
       matrix:
         version:
           - ""
-          - v22.3.17
-          - v22.2.11
```

Comment:
Did we deprecated 22.2.X?

Reply:
Yes and no. I haven't removed any of the version checking yet, but it's not supported anymore.